examples/models/data_tables_server.py

```diff
@@ -23,10 +27,14 @@ def __init__(self):
         self.class_filter = None
 
         self.source = ColumnDataSource()
+        # A bit hacky, but now we have an index column that will carry through our slicing
+        self.df = DataFrame.from_dict(ColumnDataSource.from_df(mpg))
+        self.df.set_index('index', inplace=True)
         self.update_data()
-
-        self.document.add_root((self.create()))
-        self.session = push_session(self.document)
+        self.source.on_change('data', self.update_from_table_change)
+        # XXX: I haven't had much luck running the server in this style
+        # self.document.add_root((self.create()))
+        # self.session = push_session(self.document)
```

Comment:
@bgyarfas you definitely have something odd configured locally I think. FWIW I just tried locally from master and it worked as expected:

<img width="790" alt="screen shot 2016-09-09 at 10 50 51 am" src="https://cloud.githubusercontent.com/assets/1078448/18393539/5fb613d4-767b-11e6-9301-8774cbb396ea.png">


<img width="1046" alt="screen shot 2016-09-09 at 10 50 59 am" src="https://cloud.githubusercontent.com/assets/1078448/18393542/648ee41c-767b-11e6-91bd-9c3a0289b14c.png">

<img width="1356" alt="screen shot 2016-09-09 at 10 51 06 am" src="https://cloud.githubusercontent.com/assets/1078448/18393547/67de8384-767b-11e6-9b69-db9a9b4c371e.png">


So I think my preferred way forward is simply to fix up the merge conflict with master, revert the changes to this example, and then merge the PR. @birdsarah or I can test the PR before merging. We could open a separate issue for the problems you are experiencing.